source/index.rst

```diff
@@ -11,7 +11,184 @@
 FIRST Robotics Competition Control System
 ===========================================
 
-Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation! An overview of the changes from 2020 to 2021 is available on the :ref:`New for 2021 <docs/yearly-overview/yearly-changelog:New for 2021>` document.
+Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation! This site will contain everything you need to know for building and programming a competition robot!
```

Comment:
```suggestion
Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation! This site contains everything you need to know for programming a competition robot!
```

"will contain" is future tense.  I like using present here.  As is sounds like things are incomplete.